utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -419,6 +419,10 @@ int main(int argc, char **argv)
     if(testCase == 5)
         CHECK_RETURN_STATUS(hipHostMalloc(&d_interDstPtr, srcDescPtr->strides.nStride * srcDescPtr->n * sizeof(Rpp32f)));
 
+    Rpp32f *alpha;
```

Comment:
Assign a nullptr for alpha here during declaration

Reply:
Done